daemon/analytics.h

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_ANALYTICS_H
+#define NETDATA_ANALYTICS_H 1
+
+#include "../daemon/common.h"
+
+/* Max number of seconds before the META analytics is sent */
+#define ANALYTICS_MAX_SLEEP_SEC 20
+
+/* Maximum number of hits to log */
+#define ANALYTICS_MAX_PROMETHEUS_HITS 255
+#define ANALYTICS_MAX_SHELL_HITS 255
+#define ANALYTICS_MAX_JSON_HITS 255
+
+#define NETDATA_PLUGIN_HOOK_ANALYTICS \
+    { \
+        .name = "ANALYTICS", \
+        .config_section = NULL, \
+        .config_name = NULL, \
+        .enabled = 0, \
+        .thread = NULL, \
+        .init_routine = NULL, \
+        .start_routine = analytics_main \
+    },
+
+struct analytics_data {
+    char *NETDATA_CONFIG_STREAM_ENABLED;
```

Comment:
Why are you using uppercase variable? I remember we are using this pattern for constants, but I do not remember to see variables in uppercase.

Reply:
Hi Thiago! Yes, you are right. I was likely carried away because they mapped to env variables which are uppercase... Changed with https://github.com/netdata/netdata/pull/10887/commits/e57d39cf78eab69e61aedbe5d51692203f7d0112